connector_algolia/views/se_backend_algolia.xml

```diff
@@ -18,13 +18,14 @@
                         </group>
                     </group>
                     <group name="index" string="Index">
-                        <field name="index_ids" nolabel="1">
+                        <field name="index_ids" nolabel="1" 
+                               context="{'backend_id': active_id, 'backend_model': active_model}">
                             <tree string="Index" editable="bottom">
-                                <field name="name"/>
                                 <field name="lang_id"/>
                                 <field name="model_id"/>
                                 <field name="exporter_id" context="{'form_view_ref': 'connector_algolia.view_ir_exports'}"/>
                                 <field name="batch_size"/>
+                                <field name="name" readonly="1"/>
```

Comment:
@florian-dacosta has got one

Reply:
@lmignon The goal of this PR is to compute the name of the index automatically, in order to avoid error.
Indeed, the format of the index name needs to follow a specific pattern and it is very easy to do it wrong, especially for a not-technical user.
We wanted to avoid to put the field as compute, as we do not know if there is a reason it should not be, maybe in generic se backend.

But, if we leave the field editable, it does not change the source problem, that we can put any wrong value into the field... Just using a on change without any readonly does not seem very strong to me...
+, as it is the first field on the view, we tend to fill this field first if it is editable.

For which reason we would want to put a different value in this field?
I am not espacially against it, you know the module better than me.
If we do not, then maybe we should put a python contraints that checks the format? (check that words are separated by '_' and end with the lang code?

Reply:
@florian-dacosta I've no problem with the fact that we enforce some rules for the field name. IMO an onchange + readonly = compute. I've in mind that readonly fields are not posted by odoo when pressing the save button (without the read_only_bypass addon). Moreover, the onchange is not applied to records created by xml data.
We should set the field as computed by default. If a future specialization needs to allow the user to freely set the name, this kind of need can easily be implemented by adding a new field with an inverse method or an override of the compute method to take the value of this new field if specified.
With this change, we are sure that the rule is enforced and the name is computed also for xml data.  
If I understand, one of the problem you try to solve with readonly=True is

Reply:
@lmignon 
Well yes, a computed field is better, I agree.
@bealdav Since Laurent agree for a computed field, can you change the field this way.